pkg/ingester/ingester_v2.go

```diff
@@ -455,14 +475,15 @@ func NewV2(cfg Config, clientConfig client.Config, limits *validation.Overrides,
 	i := &Ingester{
 		cfg:           cfg,
 		clientConfig:  clientConfig,
-		metrics:       newIngesterMetrics(registerer, false, cfg.ActiveSeriesMetricsEnabled),
 		limits:        limits,
 		chunkStore:    nil,
 		usersMetadata: map[string]*userMetricsMetadata{},
 		wal:           &noopWAL{},
 		TSDBState:     newTSDBState(bucketClient, registerer),
 		logger:        logger,
+		ingestionRate: newEWMARate(0.2, cfg.RateUpdatePeriod),
```

Comment:
[question, cause I'm unsure] Default `RateUpdatePeriod` is 15s. I'm wondering if it's a good setting for this use case. Basically we could get 15x spike in 1 single second (and we'll accept it) but then we'll reject any sample for the next 14s, right?

Reply:
You're right. I suggest we don't make this one configurable, and instead use hard-coded 1s tick. WDYT?